service/src/main/java/bio/terra/landingzone/common/VirtualNetworkLinkResourceHelper.java

```diff
@@ -0,0 +1,26 @@
+package bio.terra.landingzone.common;
+
+import bio.terra.landingzone.library.landingzones.definition.ArmManagers;
+
+public class VirtualNetworkLinkResourceHelper {
+  private VirtualNetworkLinkResourceHelper() {}
+
+  public static void delete(ArmManagers armManagers, String resourceId) {
+    var idParts = resourceId.split("/");
```

Comment:
This is pretty awkward. Curious if there's a more directed API to get at the identifiers without having to tokenize a string. Probably worth looking into if the related bug doesn't fixed soon-ish.

Reply:
The SDK team confirmed issue and mentioned that fix will be available around mid-August. https://github.com/Azure/azure-sdk-for-java/issues/36152